cli/config/compose/profiles/ingest-manager/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       - xpack.security.authc.api_key.enabled=true
       - ELASTIC_USERNAME=elastic
       - ELASTIC_PASSWORD=changeme
-    image: "docker.elastic.co/observability-ci/elasticsearch:${stackVersion:-7.9.0}"
+    image: "docker.elastic.co/elasticsearch/elasticsearch:${stackVersion:-7.9.0}"
```

Comment:
@mdelapenya do you think we can make this dynamic to 'latest avialable of the 7.9.X line' as opposed to hard-coded to the 7.9.0 Beta build?  It is rather implied that its pulling .X latest else we can create a new branch '7.9.0' in e2e-testing ?

Reply:
If we have a "moving target" for that version, I'd hardcoded here. I chose 7.9.0 or the released one. We can always override for local or CI specific runs. If we find ourselves replacing it many times, it's the time to update the default value.